README.md

```diff
@@ -73,4 +73,33 @@ logging_config:
         x-honeycomb-team: "YOUR_API_KEY"
 ```
 
-The gRPC metadata can also be specified on the command line, with `--otlp-metadata x-honeycomb-team=YOUR_API_KEY`.
\ No newline at end of file
+The gRPC metadata can also be specified on the command line, with `--otlp-metadata x-honeycomb-team=YOUR_API_KEY`, or through the environment variable `OTLP_METADATA`.
+
+## OpenTelemetry Metrics
+
+Application-level metrics from the server can be exported to one of the following services.
+
+### Prometheus
+
+When the Prometheus exporter is enabled, a server will listen on port 9464 for metrics scrape requests. Prometheus must be configured to scrape the server, either manually or via an auto-discovery mechanism. Compile `janus_server` with the `prometheus` feature enabled, and add the following to the configuration file.
+```yaml
+metrics_config:
+  exporter:
+    prometheus:
+```
+
+### Honeycomb
+
+Honeycomb also supports OpenTelemetry-formatted metrics, though only on the Enterprise and Pro plans. Compile `janus_server` with the `otlp` feature enabled, and add the following section to the configuration file. Note that the OTLP/gRPC exporter will push metrics at regular intervals.
+
+```yaml
+metrics_config:
+  exporter:
+    otlp:
+      endpoint: "https://api.honeycomb.io:443"
+      metadata:
+        x-honeycomb-team: "YOUR_API_KEY"
+        x-honeycomb-dataset: "YOUR_METRICS_DATASET"
+```
+
+The command line flag `--otlp-metadata` or environment variable `OTLP_METADATA` may alternately be used to supply gRPC metadata for the metrics exporter, as with the tracing exporter above.
```

Comment:
Q: is it possible we'll want to specify different OTLP metadata to the metrics vs the logging config? Or alternatively, a reason to believe we _won't_ want to specify different metadata?